src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerTestingControllerTest.java

```diff
@@ -48,6 +49,7 @@ public void setup() {
     }
 
     @Test
+    @Ignore("It is failing at the moment")
```

Comment:
We should only ignore if there is a valid reason as this was passing before. What is the reason this fails and is there a separate ticket to address?